views/list.hbs

```diff
@@ -4,16 +4,30 @@
 
 <body>
     <h2>Demos</h2>
+    <p>The main endpoint is '/list/position' The defaults will search 24 hours worth of data for list positions 1, 2,
```

Comment:
Might be worth having a url builder like the one we have for the facets, with the available lists in a dropdown/datalist